server/chat-commands.js

```diff
@@ -3029,8 +3029,8 @@ const commands = {
 				this.sendReply("Formats have been hot-patched.");
 			} else if (target === 'loginserver') {
 				FS('config/custom.css').unwatch();
-				Chat.uncache('./server/loginserver');
-				global.LoginServer = require('./loginserver').LoginServer;
+				Chat.uncacheDir('./.server-dist/loginserver');
```

Comment:
`uncacheDir` seems odd here because `loginserver` is not a directory?

Reply:
Yeah, that was a mistake on my end.